src/docs/frontend/pull-request-previews.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Pull Request Previews
+---
+
+## What are preview builds?
+
+When you open a pull request on GitHub in the sentry repository, Vercel can automatically build and deploy the frontend code in your branch into a preview URL. This allows you to see the feature or fix as it would be in production.
```

Comment:
```suggestion
When you open a pull request on GitHub in the sentry repository, Vercel can automatically build and deploy the frontend code in your branch into a preview URL. This allows you to see the feature as it would be in production.
```

Reply:
```suggestion
When you open a pull request on GitHub in the sentry repository, Vercel can automatically build and deploy the frontend code in your branch into a preview URL. This allows you to see the change as it would be in production.
```